### ISSUE TITLE
subtle stuttering during playback

### ISSUE THREAD

**Issue by HarHarLinks** [jellyfin/jellyfin-media-player#204]
**Describe the bug**
There is very very slight stuttering during playback, which does not happen when I use mpv directly. The only other difference is that one method also goes via Jellyfin server (docker, direct playback only), while "directly" is using a samba share mounted with cifs. Jellyfin and NAS are both on the same machine in the LAN, with GbE.

The stutter happens in intervalls of 10-20 seconds at most, but not always, and only drops 1-5 frames or so I'd say. Since I am watching high quality 4K content, I can't exclude this being a peak bitrate issue in either Jellyfin server or client buffer. It does happen for example at 16Mb/s (mean) bitrate, which according to plotbitrate has frequent peaks over 30Mb/s.

![example](https://user-images.githubusercontent.com/2803622/149662362-7debff69-fa62-4ab9-afa9-c859f260e7ea.png)

My primary display is 170Hz (freesync/VRR) 1440p, which I use for viewing. My windowmanager is sway, and even though the log calls the display "XWAYLAND0", the window is running in native wayland mode. Though perhaps it appears later as `KonvergoWindow.cpp @ 685 - Screen 0 QScreen(0x55a057f4b430, name="DP-1") QRect(1920,0 2560x1440)`, so I'm not sure.
I am also using optical Dolby SPDIF bitstreaming and pipewire (but it might use ALSA directly, not sure).

**To Reproduce**
Play back.

![image](https://user-images.githubusercontent.com/2803622/149659592-b2f4be69-9885-4d19-acaa-8fa3f07e9233.png)

**Expected behavior**
0 stutter.

**Log**

**Desktop (please complete the following information):**
 - OS: arch
 - Version 1.6.1-1
 - Installation Method https://aur.archlinux.org/packages/jellyfin-media-player/

**Additional context**

**Comment by sevenrats** [comment]
i haven't been able to reproduce this issue. is it still a problem?

**Comment by HarHarLinks** [comment]
Yes it is, but I have no idea how to nail it down. Happy to answer any questions...

**Comment by krojew** [comment]
@sevenrats I can confirm this is still happening and I have 100% reproducible videos. All 4k HDR. The issue doesn't occur when playing files directly.

**Comment by krojew** [comment]
What I can also add is the stutter happens on camera shot changes which suggest stutter on keyframes.

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
